Source/UI/Validator/PhoneNumberValidator.swift

```diff
@@ -1,11 +1,39 @@
 import Foundation
+import PhoneNumberKit
+import Checkout
 
 class PhoneNumberValidator: Validator {
     private let validator = PhoneValidator()
+    private let phoneKit = PhoneNumberKit()
+    private let validCharacterSet: CharacterSet = {
+        var validInputs = "+ "
```

Comment:
is the prefix always required ?

Reply:
It is not required input, but it is valid input. The formatter will introduce it if not provided by user.